integration-tests/eks-addon/validateResources_test.go

```diff
@@ -0,0 +1,224 @@
+package eks_addon
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+
+	arv1 "k8s.io/api/admissionregistration/v1"
+	appsV1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbacV1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"testing"
+)
+
+const NAMESPACE = "amazon-cloudwatch"
+
+func TestK8s(t *testing.T) {
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Printf("error getting user home dir: %v\n", err)
+		os.Exit(1)
+	}
+	kubeConfigPath := filepath.Join(userHomeDir, ".kube", "config")
+	fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+
+	kubeConfig, err := clientcmd.BuildConfigFromFlags("", kubeConfigPath)
+	if err != nil {
+		fmt.Printf("Error getting kubernetes config: %v\n", err)
+		os.Exit(1)
+	}
+
+	clientset, err := kubernetes.NewForConfig(kubeConfig)
+
+	if err != nil {
+		fmt.Printf("error getting kubernetes config: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Validating the "amazon-cloudwatch" namespace creation as part of EKS addon
```

Comment:
nit: would be good to have print statements to say what we are validating at each step

Reply:
Will do in next PR